Changelogs for Digital Twin Manager Microservice 1024.3.0 (#4373)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1024-3-0-marked-the-rest-api-endpoint-to-get-opposite-assets-as-deprecated.md

---
date: ""

> title: "Marked the REST API endpoint to get opposite assets as deprecated"

Reviewer: ```suggestion
title: "Marked the REST API endpoint for getting opposite assets as deprecated"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2681"
version: "1024.3.0"
---

> A newer, higher-performance variant of the endpoint to query opposite assets was introduced in version 1024.1.0. To align with this improvement and maintain a consistent API, the original endpoint variant is now deprecated.

Reviewer 2: Can we add 1 line explaining what is query opposite assets as there is no other changelog explaining that

Reviewer 3: @mvoigt-sag, could you please provide the query info?

Reviewer 4: > [@mvoigt-sag](https://github.com/mvoigt-sag), could you please provide the query info?

but shouldnt that be mentioned in another ChangeLog? e.g. https://github.com/Cumulocity-IoT/c8y-docs/blob/2dac1ee2f9b2b1eefebafe6a09770375c91a9eb9/content/change-logs/application-enablement/dtm-1024-1-0-added-rest-endpoint-to-get-linked-assets-for-a-given-device.md?plain=1#L28

Reviewer 2: Thanks Michael, I’ve found the relevant changelog now. The existing changelog is good then — I just wasn’t able to find it earlier.
https://cumulocity.com/docs/change-logs/#dtm-1024-1-0-added-rest-endpoint-to-get-linked-assets-for-a-given-device

> A newer, higher-performance variant of the endpoint to query opposite assets was introduced in version 1024.1.0. To align with this improvement and maintain a consistent API, the original endpoint variant is now deprecated.

Reviewer: ```suggestion
A newer, high-performance variant of the endpoint for querying opposite assets was introduced in DTM version 1024.1.0. To align with this improvement and maintain a consistent API, the original endpoint variant is now deprecated.
```
